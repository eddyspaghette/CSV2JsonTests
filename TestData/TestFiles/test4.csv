,ぇ,ご,₣,ψ,😵
ド,え,さ,℀,ↂ,
ツ,ぃ,,せ,℟,▓
ヅ,い,け,₠,Å,○
ミ,ぅ,げ,₡,,☝
デ,う,,₢,Δ,⭐
ッ,,が,ざ,℁,⏳
