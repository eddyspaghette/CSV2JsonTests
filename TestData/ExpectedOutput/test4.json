[
 {
   "FIELD1": "",
   "FIELD2": "„Åá",
   "FIELD3": "„Åî",
   "FIELD4": "‚Ç£",
   "FIELD5": "œà",
   "FIELD6": "üòµ"
 },
 {
   "FIELD1": "„Éâ",
   "FIELD2": "„Åà",
   "FIELD3": "„Åï",
   "FIELD4": "‚ÑÄ",
   "FIELD5": "‚ÜÇ",
   "FIELD6": ""
 },
 {
   "FIELD1": "„ÉÑ",
   "FIELD2": "„ÅÉ",
   "FIELD3": "",
   "FIELD4": "„Åõ",
   "FIELD5": "‚Ñü",
   "FIELD6": "‚ñì"
 },
 {
   "FIELD1": "„ÉÖ",
   "FIELD2": "„ÅÑ",
   "FIELD3": "„Åë",
   "FIELD4": "‚Ç†",
   "FIELD5": "‚Ñ´",
   "FIELD6": "‚óã"
 },
 {
   "FIELD1": "„Éü",
   "FIELD2": "„ÅÖ",
   "FIELD3": "„Åí",
   "FIELD4": "‚Ç°",
   "FIELD5": "",
   "FIELD6": "‚òù"
 },
 {
   "FIELD1": "„Éá",
   "FIELD2": "„ÅÜ",
   "FIELD3": "",
   "FIELD4": "‚Ç¢",
   "FIELD5": "Œî",
   "FIELD6": "‚≠ê"
 },
 {
   "FIELD1": "„ÉÉ",
   "FIELD2": "",
   "FIELD3": "„Åå",
   "FIELD4": "„Åñ",
   "FIELD5": "‚ÑÅ",
   "FIELD6": "‚è≥"
 }
]